server/monitor/src/main/resources/org/apache/accumulo/monitor/resources/js/tservers.js

```diff
@@ -161,8 +161,39 @@ function clearDeadTServers(server) {
 }
 
 /**
- * Generates the tserver table
+ * Generates the tserver table and highlights rows that have recovering tservers
  */
 function refreshTServersTable() {
-  if(tserversList) tserversList.ajax.reload(null, false ); // user paging is not reset on reload
+  if (tserversList) tserversList.ajax.reload(null, false); // user paging is not reset on reload
+
+  getRecoveryList().then(function () {
+
+    // reset background for all rows
+    const table = $('#tservers').DataTable();
+    table.rows().every(function (index) {
+      $(table.row(index).node()).css('background-color', '');
+    });
+
+    const recoveryList = []
+    JSON.parse(sessionStorage.recoveryList).recoveryList.forEach(entry => {
+      recoveryList.push(entry.server);
+    });
+
+    if (recoveryList.length === 0) {
+      $('#recovery-caption').text('');
```

Comment:
You may need to hide the span as well if you don't want to see the blank space.

Reply:
And then show it when there is recovery happening.

Reply:
Yea good idea. Maybe i can just leave the text within the span and then `.hide()` and `.show()` the span when needed instead of changing the text.

Reply:
Addressed in [0ddc643](https://github.com/apache/accumulo/pull/2663/commits/0ddc643adcd20c1133da6c48e51e6c0a3729c298)